### PR TITLE
Document subscriptions

### PR DESCRIPTION
**Pull request by tobias-tengler**
Documents subscriptions and improves the Get Started document of HotChocolate.

### CODE REVIEW

**Comment by michaelstaib** [comment]
looks already very good!

File: website/src/docs/hotchocolate/defining-a-schema/operations.md [hunk 1]
```diff
@@ -279,38 +239,38 @@ public class Startup
...
-                  book: Book
-                }
+                # ...
 
                 type Mutation {
-                  addNewBook(input: AddNewBookInput): AddNewBookPayload
+                  addBook(input: BookInput): Book
+                }
+
+                type BookInput {
```
Comment (tobias-tengler) [inline website/src/docs/hotchocolate/defining-a-schema/operations.md:263]: I think we should showcase how we would bind the BookInput to the Book CLR type? I tried `BindComplexType<Book>(c => c.To("BookInput"))` but had no success. 

How would I do it in this case?

File: website/src/docs/hotchocolate/defining-a-schema/operations.md [hunk 2]
```diff
@@ -370,15 +330,242 @@ public class DefaultTransactionScopeHandler : ITransactionScopeHandler
...
+```csharp
+public class SubscriptionType : ObjectType
+{
+    protected override void Configure(IObjectTypeDescriptor descriptor)
+    {
+        descriptor
+            .Field("onBookAdded")
+            .Type<PersonType>()
+            .Resolve(context => context.GetEventMessage<Person>())
+            .Subscribe(async context =>
```
Comment (tobias-tengler) [inline website/src/docs/hotchocolate/defining-a-schema/operations.md:398]: This code works, but is this how you would do it? I haven't used subscriptions like this before...

**Review by tobias-tengler — COMMENTED** [review]
Have I forgotten anything important, that's relevant for subscriptions?

File: website/src/docs/hotchocolate/defining-a-schema/operations.md (hunk 1 above)
Comment (michaelstaib) [reply website/src/docs/hotchocolate/defining-a-schema/operations.md:263]: I will check.

**Review by michaelstaib — APPROVED** [review]

**Pull request merged**